packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -48,13 +48,14 @@ class Layout extends React.Component {
             {intl.formatMessage({ id: 'Terra.Consumer.Layout.skipcontent' })}
           </a>
         </div>
-        <div className={cx('layout', customProps.className)} {...customProps}>
-          <Nav
-            {...nav}
-            isMobileNavOpen={this.state.isMobileNavOpen}
-            onRequestClose={this.toggleNav}
-          />
-          <main id="main-container" className={cx('main-container', this.state.isMobileNavOpen && 'nav-open')}>
+        <div className={cx('layout', { open: this.state.isMobileNavOpen }, customProps.className)} {...customProps}>
+          <nav className={cx('nav')}>
```

Comment:
neat, i wasn't aware of the nav html component. I'm curious if we should be using this tag in the <Nav /> component itself and this here should just be a div for the navContainer?

Reply:
The mian reason I like it here is because its sibling is `<main>` so it reads better here...